src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -396,6 +398,29 @@ function MultiCombobox<Item>({
               isOpen={isOpen && results.length > 0}
               {...getMenuProps()}
             >
+              {isOpen && canClearAllAfter && value.length >= canClearAllAfter && (
```

Comment:
had to move this to be nested in the <Menu /> component so you could select it without it closing the dropdown.